models/fair_location.coffee

```diff
@@ -2,10 +2,9 @@ _ = require 'underscore'
 sd = require('sharify').data
 Backbone = require 'backbone'
 Relations = require './mixins/relations/location.coffee'
+Location = require './location.coffee'
 
-module.exports = class FairLocation extends Backbone.Model
-
-  _.extend @prototype, Relations
+module.exports = class FairLocation extends Location
```

Comment:
Added inheritance chain here in order to reuse the `singleLine` formatting method. This echoes our Force data model.